installer.sh

```diff
@@ -91,19 +91,32 @@ case "$ID" in
                 NEED_BUILD_TOOLS=1
                 CENTOS7_TSS_FLAGS="--enable-esapi=no --disable-doxygen-doc"
             ;;
-            8*)
+            8*|9*)
                 echo "${ID} ${VERSION_ID} selected."
                 PACKAGE_MGR=$(command -v dnf)
                 NEED_EPEL=1
                 PYTHON_PREIN="python3 python3-devel python3-setuptools python3-pip"
-                PYTHON_DEPS="gcc gcc-c++ openssl-devel python3-yaml python3-requests swig python3-cryptography wget git python3-tornado python3-zmq python3-gpg python3-psutil"
+                PYTHON_DEPS="gcc gcc-c++ openssl-devel python3-yaml python3-requests python3-cryptography wget git python3-tornado python3-zmq python3-gpg python3-psutil"
                 if [ "$(uname -m)" = "x86_64" ]; then
                     PYTHON_DEPS+=" efivar-libs"
                 fi
+                case "${VERSION_ID}" in
+                8*)
+                    PYTHON_DEPS+=" swig"
+                ;;
+                esac
                 BUILD_TOOLS="git wget patch libyaml openssl-devel libtool make automake m4 libgcrypt-devel autoconf libcurl-devel libstdc++-devel dbus-devel libuuid-devel json-c-devel autoconf-archive"
-                #TPM2_TOOLS_PKGS="tpm2-tss tpm2-tools" TODO: still on 3.1.1 tpm2_tools
-                NEED_BUILD_TOOLS=1
-                NEED_PYTHON_DIR=1
+                case "${VERSION_ID}" in
+                8*)
+                    NEED_BUILD_TOOLS=1
+                    NEED_PYTHON_DIR=1
+                ;;
+                9*)
+                    NEED_BUILD_TOOLS=0
+                    NEED_PYTHON_DIR=0
```

Comment:
`NEED_PYTHON_DIR=1`? This will create `/usr/local/lib/python3.6/site-packages/` later, that seems unrelated with `tpm2-tools`

Reply:
`NEED_PYTHON_DIR=1` was there before for RHEL 8.x, so I am not changing this fro RHEL 8.x.

Reply:
Is it OK to change it on 9?

Reply:
I think it is fine to change it, because I `NEED_PYTHON_DIR` was specific to Python 3.6 and RHEL 9 ships with 3.9

Reply:
ACK!